docs/data_management/data_responsibilities.md

```diff
@@ -0,0 +1,78 @@
+# Data Access Authorization and Responsibility
+
+Data access authorization and responsibility are critical aspect of managing and securing research data and resources. It ensures that only authorized individuals have access to specific data, maintaining security, compliance, and operational efficiency.
+
+## Shared Allocation
+
+A shared allocation is owned by a PI of a Lab or director of core facility. It is designed for sharing research data among lab members, and collaborators where permissions and access control are typically managed by the PI/director or designated administrators/manager.
+
+PIs/Core directors and their Lab members are responsible for overseeing and managing the allocations, including granting access to specific folders. However, Research Computing may provide support in certain cases. For example, if a folder becomes "locked" (i.e., no group members can change its permissions or access it), the the owner of the allocation or the folder should submit a request for us to fix the issue. In addition, if you need assistance configuring or reconfiguring permissions, we can provide support as a convenience. Simply send us a request via <support@listserv.uab.edu>.
+
+{{ read_csv('data_management/res/cheaha_project_directory.csv', keep_default_na=False) }}
+
+## Individual Allocation
+
+An individual allocations is intended for personal or individual use and is available to all UAB affiliated individuals or UAB employee's sponsored Collaborator. It is tied to the individual’s email and provide 5 TB of home/user directory on Cheaha and additional 5 TB of LTS allocation.
```

Comment:
Plural confusion. Do you mean "An individual allocation is" or "Individual allocations are"?